compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -321,7 +322,8 @@ void AMDAIETileAndFusePass::runOnOperation() {
   funcOp->walk<WalkOrder::PostOrder, ReverseIterator>([&](TilingInterface op) {
     // Find the next consumer op if it does not have loops OR if it is a
     // linalg.copy op.
-    if (op.getLoopIteratorTypes().empty() || isa<linalg::CopyOp>(op))
+    if (op.getLoopIteratorTypes().empty() || isa<linalg::CopyOp>(op) ||
+        isa<tensor::UnPackOp>(op))
```

Comment:
Can we create a static utility, say, `opToSkip(op)` (I'm not good with names) and put both `linalg.copy` and `tensor.unpack`?
And accordingly we should update the preceding comment.

Reply:
Done